.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,33 @@
+{
+	"name": "ngrx-traits",
+	"build": {
```

Comment:
Hey Valerio, first, thanks for this is looking great. I have a few questions; sorry if they might be silly I'm new to dev containers.  
- I just wondering if we could use a prebuilt image like ngrx guys use instead of our own docker build.
in their devcontainer see
https://github.com/ngrx/platform/blob/0ae21c92451dd9b9c533da4bca0dc61e6d8f4fca/.devcontainer/devcontainer.json
They use mcr.microsoft.com/devcontainers/typescript-node:0-18-bullseye

Also, I was  thinking, It will be nice to have a little md about it, ngrx has it in the  contributing section
https://github.com/ngrx/platform/blob/0ae21c92451dd9b9c533da4bca0dc61e6d8f4fca/CONTRIBUTING.md
Will this config we are doing work as well with github codespaces?, sounds like are the same or similar

Reply:
Hey Gabriel! 
This configuration is based on a prebuilt image, the same you have linked to me. Please see Dockerfile in .devcontainer folder. I use a custom build because DC require some extra tweaks in order to run e2e test and more.

Furthermore, do you ask me to add a _CONTRIBUTING.md_ to this project?

Reply:
@valeriocomo , sorry for taking so long to get back to you; I have been busy. Yeah, you could create a CONTRIBUTING.md I don't mind, if you want maybe use as a base the ngrx one or if you prefer a simple md inside the  /docs folder like devcontainer.md, so I can link it to the signals docs